biggerquery/beam_manager.py

```diff
@@ -0,0 +1,205 @@
+import avro.schema
+from apache_beam.io import BigQueryDisposition
+
+import avro.schema
+import apache_beam as beam
+from apache_beam.io.gcp.internal.clients import bigquery
+from apache_beam.options.pipeline_options import \
+    PipelineOptions, SetupOptions, StandardOptions, GoogleCloudOptions, WorkerOptions
+
+
+class BeamManager:
+    """
+    Used for read/write operations. It is base class for further operations.
+    """
+    def __init__(self):
+        self.pipeline = None
+
+    def read_from_big_query(self, dataset, project_id, standard_sql, query):
+        return beam.io.Read(beam.io.BigQuerySource(
+            dataset=dataset,
+            project=project_id,
+            use_standard_sql=standard_sql,
+            query=query
+        ))
+
+    def read_from_avro(self, file_source_path):
+        return beam.io.ReadFromAvro(file_source_path)
+
+    def write_to_avro(self, file_output_path, schema):
+        return beam.io.WriteToAvro(
+            file_output_path,
+            avro.schema.parse(schema),
+            num_shards=1,
+            shard_name_template='S_N',
+            file_name_suffix='.avro')
+
+
+class TemplatedBeamManager:
+    """
+    Decorator that resolves table names/all kind of variables.
+    """
+
+    def __init__(self, beam_manager, internal_tables, external_tables, run_datetime, extras):
```

Comment:
change run_datetime to runtime everywhere

Reply:
Look now please.